doc/source/servers/huggingface.md

```diff
@@ -70,3 +71,37 @@ spec:
     replicas: 1
 ```
 
+## Custom Model Example
+
+You can deploy a custom HuggingFace model by providing the location of the model artefacts using the `modelUri` parameter.
```

Comment:
Not a big deal, but to avoid confusion you could refer to the "`modelUri` field"?

Reply:
Oh, my bad, that's what I meant!